Content.Server/Forensics/Components/ForensicsComponent.cs

```diff
@@ -11,5 +11,17 @@ public sealed partial class ForensicsComponent : Component
 
         [DataField("dnas")]
         public HashSet<string> DNAs = new();
+        
+        /// <summary>
+        ///     Chance of transfering DNA from this entity to spawned one after this this being destroyed
+        /// </summary>
+        [DataField("dnaTransferChanceFromDestroyed")]
+        public float DNATransferChanceAfterDestroy = 0f;
+
+        /// <summary>
+        ///     Chance of transfering fibers and fingerprints from this entity to spawned one after this this being destroyed
+        /// </summary>
+        [DataField("restOfTransferChanceFromDestroyed")]
+        public float RestOfTransferChanceAfterDestroy = 0f;
```

Comment:
Please use more descriptive field names and this means nothing without further context.